manticore/platforms/linux.py

```diff
@@ -826,7 +845,10 @@ def load(self, filename):
                 continue
             interpreter_filename = elf_segment.data()[:-1]
             logger.info('Interpreter filename: %s', interpreter_filename)
-            interpreter = ELFFile(file(interpreter_filename))
+            try:
+                interpreter = ELFFile(file(interpreter_filename))
```

Comment:
Check with the `python3` branch; this might have been changed to use `open`.

Reply:
Ok